pos_partner_deselection/doc/index.rst

```diff
@@ -1,25 +1,29 @@
-==================
- Deselect Partner
-==================
+=====================
+ Partner Deselection 
+=====================
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
 
 Configuration
 =============
 
-* Go to ``Point of Sale >> Configuration >> Point of Sale``
+* Go to a ``Point of Sale`` 
```

Comment:
We have a special formatting for menus: http://xn--v1aaa.xn--b1a9bba.xn--p1ai/en/latest/dev/docs/rst-requirements.html#references-to-menu

Reply:
fixed